panel/src/panel/content.js

```diff
@@ -109,8 +143,9 @@ export default (panel) => {
 		 * @param {Object} fields
 		 * @param {any} value
 		 */
-		set(fields) {
-			panel.app.$store.dispatch("content/update", [null, fields]);
+		set(values) {
+			this.changes = values;
+			this.save();
```

Comment:
`Todo:` I think this will leave to troubles when every input event sets the content module state and immediately triggers a save API request where subsequent request don't cancel each other out. We need some kind of throttling/debouncing here and prevention of race conditions.

Reply:
Absolutely! I wanted to keep it very simple in the first iteration and then make it more sophisticated step by step.